Makefile

```diff
@@ -27,3 +27,10 @@ presentation.pdf: presentation/presentation.tex
 	cd presentation && pdflatex presentation.tex
 	cp presentation/presentation.pdf .
 
+fullplain.html: rules/generate.rb rules/full.txt
+fullplain.html: $(patsubst %.svg,%.png.b64,$(wildcard rules/texfiles/*.svg))
+fullplain.html:
+	cd rules && ruby generate.rb -T ../fullplain.html
```

Comment:
Given that [the new `ctfwsd` for the HTML handbook](https://github.com/cmukgb/ctfws-host-daemons/blob/master/inotifywait-handbook) will be polling `handbook.html` (by extension from `handbook.pdf`), do we want that to be the target here instead?

Reply:
That seems like a good plan.